content/reference/drops/context_drop.md

```diff
@@ -0,0 +1,14 @@
+# Context Drop
+
+Allows to check does requested data loaded properly by `true`/`false` boolean statement.
```

Comment:
Noy sure about the description, it's rather for checking if response is success or not, it kinds means something similar in this case, maybe it `allows to check if any exception happened during the request`?